gnomad/utils/constraint.py

```diff
@@ -597,9 +599,9 @@ def build_plateau_models(
         1],[1,1,1]]`. Default is None.
     :param weighted: Whether to generalize the model to weighted least squares using
         'possible_variants'. Default is False.
-    :return: A dictionary of intercepts and slopes for plateau models of each
-        population. The keys for this dictionary are 'total' and 'pop'. The values for
-        'total' is a dictionary (e.g., <DictExpression of type dict<bool,
+    :return: A dictionary of intercepts and slopes of plateau models for all set and
+        for populations. The keys for this dictionary are 'total' and 'pop'. The values
```

Comment:
```suggestion
        for populations. The possible keys for this dictionary are 'total' and 'pop'. The values
```
adding "possible" since this function won't add the 'pop' key by default